app/assets/stylesheets/views/_travel-advice.scss

```diff
@@ -238,64 +230,10 @@
     }
   }
 
-
-  aside {
-    .page-navigation {
-
-      li {
-        list-style:none;
-        margin-left: 0;
-      }
-
-      @include media(mobile) {
-        li,
-        ol[start] li,
-        li a,
-        li span {
-          margin-left: 0;
-        }
-
-        li.active {
-          border-top: solid 1px #a1acb2;
-          margin: 0;
-          padding: 1em;
-        }
-
-        li a {
-          padding-left: 1em;
-        }
-
-        li span {
-          display: block;
-          padding: 0;
-          border-top: none;
-        }
-      }
-    }
-    .subscriptions  {
-      p {
-        @include core-19;
-        font-weight: 700;
-        margin: 0;
-        margin-right: 0.5em;
-
-      }
-
-      a {
-        @include core-19;
-        font-weight: 700;
-      }
-
-      padding-top: 2em;
-      clear: left;
-      float: none;
-      text-align: left;
-      width: auto;
-
-      @include media-down(mobile) {
-        padding-left: 1em;
-      }
-    }
+  .page-navigation {
+    border-bottom: 1px solid #bbb;
```

Comment:
Is there a reason for not using a standard colour for this (e.g. `$govuk-border-colour`)?

Reply:
Updated.